### DISCUSSION TITLE
Tooltip Usage in a Chrome Extension

### DISCUSSION THREAD

**Discussion by schenktobi** [themesberg/flowbite-vue#162] [Q&A]
Hi there, I followed the Installation Guideline and tried to use a Tooltip inside my Chrome Extension, which renders an iFrame into my current Page. When using the default Tooltip and hovering over the trigger element, the scrollbar moves, but i cannot see the Tooltip at all.

Are there any interferences because of the iFrame or anything similar? I dont know if im specific enough.

**Comment by cogor** [comment]
Hi, you have a very interesting case study, I've never worked with extensions before, could you put together an example so I can reproduce this problem?

**Reply by schenktobi** [reply]
Hi, if i find some time in the following days, I can try to produce a small example repository to show the Error/Bug, but i shifted aaway from using flowbite because of this problem. I dont know when I will find the Time, but I will try my best to find a slot
